base/linalg/bidiag.jl

```diff
@@ -154,17 +172,38 @@ for func in (:A_ldiv_B!, :Ac_ldiv_B!, :At_ldiv_B!)
         B
     end
 end
-for func in (:A_ldiv_Bt!, :Ac_ldiv_Bt!, :At_ldiv_Bt!)
+
+function A_ldiv_Bt!(A::Union(Bidiagonal, AbstractTriangular), B::AbstractMatrix)
+    mA,nA = size(A)
+    tmp = similar(B[1,:])
+    tmp = tmp.'
+    m, n = size(B)
+    if mA != n || nA != m
+        throw(DimensionMismatch())
+    end
+    for i = 1:size(B, 1)
+        copy!(tmp, B[i,:])
+        A_ldiv_B!(A, tmp)
+        B[i,:] = tmp.'
+    end
+    B.'
```

Comment:
Yes, I agree. I don't think we're going to get any performance benefit from including these extra "full" definitions and it's confusing. Thanks for the feedback.